web/templates/classEmma.class.php

```diff
@@ -541,6 +541,44 @@ function getClubResults($compId, $club)
 			return $ret;
 	}
 
+	function getPending($compId, $limit)
+	{
+		$ret = Array();
+		$q = "SELECT runners.Name, runners.Club, results.Time, runners.Class ,results.Status, results.DbID, results.Control ";
+		$q .= "FROM runners,results ";
+		$q .= "WHERE results.DbID = runners.DbId AND results.TavId = ". $this->m_CompId ." AND runners.TavId = ".$this->m_CompId ." AND results.Status in (9,10) ";
+		$q .= "ORDER BY runners.Class, runners.Name ";
+		$q .= "LIMIT $limit";

```

Comment:
The limit does not make much sense like this, it should be limiting the number of pending runners to select (so we don't return the full DB if queried too early) - currently it limits the number controls.